### PR TITLE
abt stack

### PR DESCRIPTION
**Pull request by ashleypittman**
- Revert "DAOS-7174 pool,container: add common label verify function (#6007)"
- DAOS-7968 test: Turn on stack protection in argobots

### CODE REVIEW

File: src/engine/init.c [hunk 1]
```diff
@@ -425,6 +424,8 @@ set_abt_max_num_xstreams(int n)
 	return 0;
 }
 
+static bool d_abt_on;
```
Comment (bfaccini) [inline src/engine/init.c:427]: Why did you introduce this ?? Did you trigger some issue (deadlock ?) due to early logging ??

**Review by bfaccini — COMMENTED** [review]
have you been able to trigger some ULT stack overflow/corruption on server side with this patch ??

File: src/engine/init.c (hunk 1 above)
Comment (ashleypittman) [reply src/engine/init.c:427]: Arbobots hits this assert if compiled with debugging without this change.
https://github.com/pmodels/argobots/blob/main/src/self.c#L135

**Comment by ashleypittman** [comment]
> have you been able to trigger some ULT stack overflow/corruption on server side with this patch ??

Yes, and the two that I've managed to trigger are both resolved so it's stable now.

**Pull request closed without merging**